dts/bindings/mtd/st,stm32l0-nv-flash.yaml

```diff
@@ -0,0 +1,17 @@
+description: |
+  ST STM32L0 family flash memory.
+
+include: st,stm32-nv-flash.yaml
+
+compatible: st,stm32l0-nv-flash
+
+properties:
+  write-block-size:
+    required: true
+    type: int
+    enum:
+      - 1
+      - 2
+      - 4
+    description: |
+      Number of bytes used in write operations.
```

Comment:
I'm curious about the absence of default here..

Reply:
Unlike the F4 series, in the L0 series, the write block size isn't configurable via a register, so I wasn't sure what value could be used as the default, if any.